doc/tools/docgram/fullGrammar

```diff
@@ -604,7 +605,7 @@ command: [
 | "Typeclasses" "Opaque" LIST1 reference
 | "Typeclasses" "eauto" ":=" debug eauto_search_strategy OPT natural
 | "Proof" "with" Pltac.tactic OPT [ "using" G_vernac.section_subset_expr ]
-| "Proof" "using" G_vernac.section_subset_expr OPT [ "with" Pltac.tactic ]
+| "Proof" "using" G_vernac.section_subset_expr "with" Pltac.tactic
```

Comment:
I don't care so much but the `Proof using` changes are unrelated to this PR (these are obviously late changes to the `Proof using` PR #13339).

Reply:
Yeah, but until we implement a CI check that `fullGrammar` and `orderGrammar` are updated, this is how we update these files.

Reply:
`make doc_gram_rsts` automatically generates `fullGrammar` from all the mlgs.  In this case, we're picking up an mlg change from another PR for which the submitter didn't run `doc_grammar`.  `fullGrammar` shouldn't be edited by hand.  It is for reference, doesn't affect the binaries.